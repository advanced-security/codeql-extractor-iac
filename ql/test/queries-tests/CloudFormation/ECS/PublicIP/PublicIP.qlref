security/CloudFormation/ECS/PublicIP.ql
