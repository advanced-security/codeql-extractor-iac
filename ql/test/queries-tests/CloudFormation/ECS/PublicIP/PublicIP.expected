| ecs.yml:61:27:61:35 | 'ENABLED' | AssignPublicIp should be "DISABLED" for ECS tasks |